railties/lib/rails/generators/actions.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require "shellwords"
```

Comment:
Is this the appropriate place for this `require`?

Reply:
Seems fine to me 👍 — maybe the generators have already required it elsewhere, but fine to keep this here too.